pages/index.js

```diff
@@ -1,15 +1,13 @@
-import  {Home}  from "../components/Home";
-import { Header} from "../components/Header";
-import { Navigation } from "../components/Navigation";
-
-
+import Main from "../components/Main";
+import Header from "../components/Header";
+import Navigation from "../components/Navigation";
 
 export default function HomePage() {
   return (
-  <div>
-    <Header />
-    <Home />
-    <Navigation />
-  </div>
-  )
+    <div>
+      <Header />
+      <Main />
+      <Navigation />
+    </div>
```

Comment:
You should move your >`<Header>` and `<Navigation>`  to your `_app.js` to have them visible on every page and not only on your homepage.

Reply:
Thank you, I did all the requested changes including the link to Vercel.